### ISSUE TITLE
Face Recoginition using VGG16

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/Face-X#324]

**Comment by arnab031** [comment]
I can do that using VGG16. But what dataset will I use for that? @akshitagupta15june

**Comment by hrithikppawar** [comment]
> I am a GSSOC21 Participant. I can do that using VGG16 in PyTorch. Can I use any dataset from kaggle for that? @akshitagupta15june

That's great, I think we can collaborate and work on many other architectures and can also work on real time face mask detection using live camera.

**Comment by HOTSONHONET** [comment]
Hello ma'am, I'm a GSSOC'21 participant and I want to know if the recognition is based on face verification or identification? I'm comfortable in both cases. Can you please assign this issue to me? If assigned I will be using tf.keras's VGG16 model to build a siamese network for this task.
